src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,78 @@
+package duke.command;
+
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+
+import java.io.IOException;
+import java.util.stream.Stream;
+
+/**
+ * Class which consists of Tasks which adds to the List
+ * Eg: Deadline, TO-DO, Event.
+ */
+public class AddCommand extends Command {
+
+
+    /**
+     * Instantiates a new Add command.
+     *
+     * @param userInput the user input
+     */
+    public AddCommand(String userInput) {
+        super(userInput);
+    }
+
+
+    /**
+     * Overwrites the execute method from Abstract class execute.
+     * Check against the user's input then pass it to its respective task class.
+     *
+     * @param storage  Deals with loading tasks from file.
+     * @param ui       Deals with interactions with the user
+     * @param taskList List containing all the tasks
+     * @throws DukeException Main exception method I have created
+     * @throws IOException   For any potential Input/Output exceptions from incorrect file
+     */
+    @Override
+    public String execute(Storage storage, Ui ui, TaskList taskList)
+            throws DukeException, IOException {
+
+        if (userInput.contains("deadline")) {
+            assert (userInput.contains("deadline")) : "userInput should contain deadline!";
```

Comment:
This assertion might be redundant, as you can only enter the `assert` portion if you successfully check whether `userInput.contains("deadline")` returns true.